src/node/quote.h

```diff
@@ -212,6 +212,7 @@ namespace ccf
   public:
     static CodeDigest get_code_id(const QuoteInfo& quote_info)
     {
+      LOG_INFO_FMT("GETTING CODE ID");
```

Comment:
If we move this log line lower, just before the return, we could log the code ID we retrieved and keep it as a useful logging line, not just for the temporary test?

Reply:
Agreed but logging that as info seems quite verbose?

Reply:
This would be once per join, once per jwt refresh when using sgx filter, and every time the JS runtime calls verify as well, so as discussed, it's too verbose.